ArtifactScript.py

```diff
@@ -1,11 +1,43 @@
+import os
 import requests
 import json
+import datetime
 
-PlaySet0r = requests.get("https://playartifact.com/cardset/00/").json()
-PlaySet1r = requests.get("https://playartifact.com/cardset/01/").json()
+global PlaySet0r
+PlaySet0r = {}
+global PlaySet1r
+PlaySet1r = {}
+global PlaySet0
+PlaySet0 = {}
+global PlaySet1
+PlaySet1 = {}
+global Expiretime
 
-PlaySet0 = requests.get(PlaySet0r["cdn_root"] + PlaySet0r["url"]).json()
-PlaySet1 = requests.get(PlaySet1r["cdn_root"] + PlaySet1r["url"]).json()
+def GetAPIData():
+
+    global PlaySet0r
+    global PlaySet1r
+    global PlaySet0
+    global PlaySet1
+    global Expiretime
+
+    PlaySet0r = requests.get("https://playartifact.com/cardset/00/").json()
+    PlaySet1r = requests.get("https://playartifact.com/cardset/01/").json()
+
+    Expiretime = PlaySet0r["expire_time"]
+
+    with open('PlaySet0.json', 'w') as f:
+        json.dump(requests.get(PlaySet0r["cdn_root"] + PlaySet0r["url"]).json(), f)
```

Comment:
I can't find any documentation on what the `'cdn_root'` attribute returns. If it's blank, then that means this line is just downloading the card set again.